test/integration.sh

```diff
@@ -52,7 +52,7 @@ sed -i \
 export LORIS_DB_CONFIG=$(pwd)/config.xml
 
 #start PHP's built in webserver
-php -S localhost:8000 -t ../htdocs ../htdocs/router.php 2>&1 > /dev/null &
+php -S localhost:8000 -t ../htdocs ../htdocs/router.php /dev/null 2>&1 > &
```

Comment:
What are you trying to do with this change?

Reply:
put the redirection of stderr to stdout in the right position. It came as an error when Travis run on. Oh wait, I have the and '>' misplaced.